src/script/lua_api/l_item.cpp

```diff
@@ -612,9 +612,12 @@ int ModApiItemMod::l_get_content_id(lua_State *L)
 	std::string name = luaL_checkstring(L, 1);
 
 	const NodeDefManager *ndef = getGameDef(L)->getNodeDefManager();
-	content_t c = ndef->getId(name);
+	IItemDefManager *idef = getGameDef(L)->getItemDefManager();
+	content_t content_id;
+	if (!ndef->getId(idef->getAlias(name), content_id))
```

Comment:
Afaik, `NodeDefManager::getId` already takes aliases into account. (Eg. `get_content_id("mese")` already returns the same number as `get_content_id("default:mese")` in current master.)
```suggestion
	if (!ndef->getId(name, content_id))
```

Reply:
You're right; I've fixed it now. Thanks.